app/controllers/application_controller.rb

```diff
@@ -1,12 +1,26 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
+  def current_ability
+    if admin_signed_in?
+      @current_ability ||= Ability.new(current_admin)
+    elsif partnering_organization_signed_in?
+      @current_ability ||= Ability.new(current_partnering_organization)
```

Comment:
Is our sign in stuff configured so that only one type of user/resource can be logged in at a time? I think by default it is possible to be signed in as both an admin and a PO which I feeeel like is not a good thing since it only gives them the privileges of an admin